### PR TITLE
Separator wrapper

### PR DESCRIPTION
**Pull request by edgarfgp**
This one was a quick one :)

### CODE REVIEW

**Comment by edgarfgp** [comment]
Simulator Screen Shot - iPhone 11 - 2022-11-15 at 20 21 43

File: src/Fabulous.Avalonia/Views/Controls/Separator.fs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+namespace Fabulous.Avalonia
+
+open Avalonia.Controls
+open Fabulous
+open Fabulous.StackAllocatedCollections.StackList
+
+type IFabSeparator = inherit IFabTemplatedControl
+
+module Separator =
+    let WidgetKey = Widgets.register<GridSplitter>()
```
Comment (TimLariviere) [inline src/Fabulous.Avalonia/Views/Controls/Separator.fs:10]: Should we call it a `GridSplitter` instead of a `Separator` to match the name in Avalonia?

Comment (edgarfgp) [reply src/Fabulous.Avalonia/Views/Controls/Separator.fs:10]: There is a Separator https://docs.avaloniaui.net/docs/controls/separator#reference . ?

Comment (edgarfgp) [reply src/Fabulous.Avalonia/Views/Controls/Separator.fs:10]: GridSplitter is a separate one?  https://docs.avaloniaui.net/docs/controls/gridsplitter#reference

Comment (TimLariviere) [reply src/Fabulous.Avalonia/Views/Controls/Separator.fs:10]: Yes, looks like it's 2 different controls:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/Separator.cs
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/GridSplitter.cs

The GridSplitter seems to be movable while Separator is not.
For this PR, I think it's better to change to `Widgets.register<Separator>`

Comment (edgarfgp) [reply src/Fabulous.Avalonia/Views/Controls/Separator.fs:10]: Ohh yeah my bad 😅 it’s playing with both

Comment (edgarfgp) [reply src/Fabulous.Avalonia/Views/Controls/Separator.fs]: Done :)

**Pull request merged**